modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +190,31 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+The Batch Job scope provides Scheduling Strategy options for controlling the
+execution of job instances.
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+state at the same time, the instances execute one at a time based on their
+creation timestamp.
++
+Note that there is no guarantee that the records will be executed in order.
+This setting does not control the execution order of records in the batch job,
+nor does it take into account the number of records each instance contains.
++
+* ROUND_ROBIN: This setting attempts to execute all available instances of a
+batch job using a round-robin algorithm to assign the available resources. It
+works for the use case in which jobs are triggered manually and you want them to
```

Comment:
`It works for the use case in which jobs are triggered manually and you want them to
run concurrently,`. -> This is wrong. It should say:

`This option is useful when you can guarantee that no job execution can have a side effect on another one. For example, this is not a good fit for data synchronization jobs. This is because the same record could have been updated in two concurrent jobs, and since their ordering is not guarantee, you might end up with a prior version of the data. However, if your batch job is just picking up new records from a database or lifting individual files from an SFTP server (and you're sure that all records are completely independent), then you can safely use this strategy to parallelize the job's execution`

Reply:
ok, cool. Pls note that the blog says this, "In the December 2013 release there was no way to configure that. By default, the batch module would try to execute all available instances by using a round robin algorithm to assign the available resources. This is fine for the use case in which jobs are triggered manually and you want them to run concurrently. But there’re other uses cases, in which you need the first job instance to finish before allowing any others to run. An example of this situation is a one-way sync between two systems." and "Round Robin: This is the behaviour we used to have in December 2013 release."

Reply:
done